nemo/core/classes/modelPT.py

```diff
@@ -1704,6 +1704,33 @@ def on_train_end(self):
         if hasattr(self, '_freeze_cfg'):
             delattr(self, '_freeze_cfg')
 
+    @classmethod
+    def from_pretrained(
+        cls,
+        model_name: str,
```

Comment:
This is not a requirement for ASR / TTS models. Should be moved up to NLPModel if such a constraint needs to be placed.

Reply:
Moved this to NLPModel. ModelPT is now unchanged